src/worldstate/routine/Gate/include/alignWithGate.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by joel on 12/05/18.
+//
+
+#ifndef PROJECT_ALIGNWITHGATE_H
+#define PROJECT_ALIGNWITHGATE_H
+
+#include "State.h"
+#include <constants.h>
+#include <gate_detect/gateDetectMsg.h>
+
+class alignWithGate : public State {
```

Comment:
Class names (and their file name) should be AllCapsCamelCase

Reply:
Refactored!